src/main/java/Chatbot/alain/GUI_Ui.java

```diff
@@ -0,0 +1,149 @@
+    private String logo = "    _     __         _     _____   ___   __\n"
+            + "   / \\   | |        / \\   |_   _| | |\\\\  | |\n"
+            + "  / _ \\  | |       / _ \\    | |   | | \\\\ | |\n"
+            + " / /_\\_\\ | |____  / /_\\_\\  _| |_  | |  \\\\| |\n"
+            + "/_/   \\_\\|______|/ /   \\_\\|_____| |_|   \\__| \n";
+
+    /**
+     * Displays a line separator.
+     */
+    public String showlines() {
+        return ("_______________________________________________\n");
+    }
+
+    /**
+     * Displays a welcome message.
+     */
+    public String showWelcome() {
+        String output = "";
+        output += this.showlines();
+        output += logo + "\n";
+        output += "Hello! I'm Alain\nWhat can I do for you?\n";
+        output += this.showlines();
+        return output;
+    }
+
+    /**
+     * Displays a goodbye message.
+     */
+    public String showGoodbye() {
+        String output = "";
+        output += this.showlines();
+        output += "Bye. Hope to see you again soon!\n";
+        output += this.showlines();
+        return output;
+    }
+
+    /**
+     * Displays an error message.
+     *
+     * @param errorMessage The error message to display.
+     */
+    public String showError(String errorMessage) {
+        String output = "";
+        output += this.showlines();
+        output += " OOPS!!! " + errorMessage + "\n";
+        output += this.showlines();
+        return output;
+    }
+
+    /**
+     * Displays the list of tasks.
+     *
+     * @param list The list of tasks to display.
+     */
+    public String showList(TaskList list) {
+        String output = "";
+        output += this.showlines();
+        output += "Here are the tasks in your list:\n";
+        for (int i = 0; i < list.size(); i++) {
+            output += " " + (i + 1) + ". " + list.getTask(i) + "\n";
+        }
+        output += this.showlines();
+        return output;
+    }
+
+    /**
+     * Displays the list of tasks containing the given keyword.
+     *
+     * @param list The list of tasks to display.
+     */
+    public String showListContainingKeyword(TaskList list) {
+        String output = "";
+        output += this.showlines();
+        output += "Here are the matching tasks in your list:\n";
+        for (int i = 0; i < list.size(); i++) {
+            output += " " + (i + 1) + ". " + list.getTask(i) + "\n";
+        }
+        output += this.showlines();
+        return output;
+    }
+
+    /**
+     * Displays a message indicating a removed task.
+     *
+     * @param removedTask The task that was removed.
+     * @param list The updated list of tasks.
+     */
+    public String showRemoveTask(Task removedTask, TaskList list) {
+        String output = "";
+        output += this.showlines();
+        output += " Noted. I've removed this task:\n"
+                + "   " + removedTask + "\n"
+                + " Now you have " + list.size() + " tasks in the list.\n";
```

Comment:
indentation of the line should be 8 spaces! so possibly adding 8 more spaces from the second + operator